doc/tutorials/using_cat_on_up2.rst

```diff
@@ -0,0 +1,157 @@
+.. _using_cat_up2:
+
+Using CAT on UP2
+################
+
+The UP Squared board `(UP2) <https://up-board.org/upsquared/specifications/>`_
+is an x86 maker board based on the Intel Apollo Lake platform, which support
+Cache Allocation Technology(CAT). With this feature, the usage of cache can be
+restricted to each VM. ACRN hypervisor can do that by giving VM a cache mask,
+so that the VM will not evict the masked cache lines, as shown in :numref:
+`cache-mask-vm`:
+
+.. figure:: images/using_cat_up2.png
+   :align: center
+   :name: cache-mask-vm
+
+   Cache usage with Cache Mask
+
+
+CAT Support on ACRN
+*******************
+
+As described at ``Intel® 64 and IA-32 Architectures Software Developer’s Manual``,
+chapter 17.17, volume 3. There are 3 steps to use CAT: detecting the CAT capability;
+setup cache mask array MSRs, which is refered as Class of Service(CLOS) array; select
+one of the CLOS array for the CPU, that will be the cache mask of the CPU.
+ACRN integrated the usage of CAT into vm configuration. The CLOS array must be defined
+at the board config source code, and the VM need to specify which CLOS to use in its vm
+config data. If the platform has CAT, the CLOS array will be write to CLOS MSRs at cpu
+init time, and the VM will set CLOS for VCPU at VM launch time.
+The details is shown in :numref:`acrn-cat-hld`:
+
+.. figure:: images/acrn_cat_hld.png
+   :align: center
+   :name: acrn-cat-hld
+
+   CAT Support on ACRN
+
+Tuning CAT at HV debug shell
+############################
+
+When at ACRN HV debug shell, user can use 'cpuid', 'wrmsr/rdmsr' debug commands to
```

Comment:
```suggestion
When at ACRN HV debug shell, user can use ``cpuid``, ``wrmsr/rdmsr`` debug commands to
```

Reply:
adding to Geoffroy's suggestion, change the beginning of the sentence to:
```
From the ACRN HV debug shell, you can use ...
```

Reply:
OK